File: Code/cpp/pattern2.cpp
#include<iostream>
using namespace std;
int main()
{
    int limit, i, j;

    cout << "Enter Limit: ";
    cin >> limit;

    for (i = limit; i > 0; i--)
    {
        for (j = i; j > 0; j--)
            cout << "* ";
        cout << "\n";
    }
    return 0;
}

File: Code/java/pattern2.java
import java.util.Scanner;

class Pattern2
{
    public static void main(String[] args)
    {
        int limit, i, j;
        String star;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Limit: ");
        limit = scanner.nextInt();

        for (i = limit; i > 0; i--)
        {
            star = "";
            for (j = i; j > 0; j--)
                star += "* ";
            System.out.println(star);
        }

        scanner.close();
    }
}